package.json

```diff
@@ -31,8 +31,8 @@
     "prestart:prod": "rimraf dist && tsc",
```

Comment:
make sure to bump the package version of this package since thats how we push updates downstream

Reply:
Whoops. Thanks!